src/styles/styles.js

```diff
@@ -2699,6 +2699,19 @@ const styles = {
         alignSelf: 'center',
     },
 
+    dragAndDropOverlay: {
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        backgroundColor: themeColors.dragAndDropBackground,
+        opacity: 0.96,
```

Comment:
IMO this opacity should be defined in the theme file

Reply:
+1

Reply:
Updated